hicommonwealth/commonwealth
# Added version history for threads and comments

## Link to Issue
Closes: #7591
Closes: #7557
Closes: #7563

## Description of Changes
- Adds comment version history for Threads and comments
<img width="642" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/14794654/5752193f-0821-4f32-a091-6c1cd2942541">


## Test Plan
- Create a thread, make sure nothing changed from master
- Create a comment, same thing
- Edit the thread, expect the dropdown to appear and allow you to change version
- Same for Comment

## Other Considerations
Unfortunately thread and comment histories were broken and was not being tracked properly. This means that only new threads and comment edits after this PR goes in will be persisted and viewable.
